.github/workflows/node.js.yml

```diff
@@ -0,0 +1,30 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: Node.js CI
```

Comment:
Suggestion: While at the moment this workflow only builds the project, we could imagine wanting to run other linting, formatting, and testing scripts (see #34). Could we rename this workflow (and the filename) to something like "Run Checks"?

Reply:
Could we also rename the file to `run-checks.yml`

Reply:
done!